tools/getrealdeps.rb

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env ruby
+# getrealdeps for Chromebrew version 1.2
+# Author: Satadru Pramanik (satmandu) satadru at gmail dot com
+require 'fileutils'
```

Comment:
Do we need to include the other libraries for color, etc.?

Reply:
no color needed.

Reply:
Good call. I adjusted that to `puts` to make things simpler.

Reply:
I see you removed `echo_error` so this should no longer be an issue.